portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Hojel's Porfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="writeSnippets()">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <center>
+          <img id="profile" src="images/profile_picture.jpg"/>
+      </center>
+      <h1 class="roboto-font">👋 Hi, I am Andrew Hojel</h1>
+      <h3 id="text" class="roboto-font">I am a </h3>
```

Comment:
this seems unfinished, maybe meant to delete it?

side note, try not to skip heading levels. https://usability.yale.edu/web-accessibility/articles/headings 
it's another accessibility thing, it's disorienting for someone who is grokking the content based on the existing headings

Reply:
In terms of design I prefer the look of the h3.... in general when deciding between aesthetic design and accessibility how should we think through the decision (I am assuming accessibility should always be the priority)?

Reply:
Yup, you definitely want to prioritize accessibility.  Luckily, the appearance can be customized with CSS, so you aren't bound to the look of the default `h2` tag.